src/components/project/MediaDetailsComp.vue

```diff
@@ -45,6 +46,7 @@ async function confirmDownload(mediaObj, fileName) {
   // URL.revokeObjectURL(url);
 
   showDownloadModal.value = false
+  logDownload({ project_id: projectStore.project_id, download_type: 'M', row_id: media_file.media_id })
```

Comment:
Can you create a constant for download as well? Then use a constant such as "DOWNLOAD_TYPES.MEDIA" instead of hardcoded letter here